src/topology/algebra/uniform_filter_basis.lean

```diff
@@ -0,0 +1,49 @@
+/-
+Copyright (c) 2021 Patrick Massot. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Patrick Massot
+-/
+
+import topology.algebra.filter_basis
+import topology.algebra.uniform_group
+
+/-!
+# Uniform properties of neighborhood bases in topological algebra
+
+This files contains properties of filter bases on algebraic that also require the theory of uniform
```

Comment:
Is there a missing noun after the word "algebraic"? Maybe "algebraic structures"?